thirdeye/thirdeye-pinot/src/main/resources/assets/css/thirdeye.css

```diff
@@ -71,6 +98,53 @@ body {
     width: auto;
     margin-top: 10px;
     text-align: right;
+    display: flex; 
+    align-items: center;
+}
+
+.te-icon {
+  color: rgba(255, 255, 255, 0.7);
```

Comment:
this one would be `app-shade(white, 7)` (I think the mixin only converts the single digit to `.7`, so only use if you don't need opacity to be set in "hundredths")